packages/cron/src/index.ts

```diff
@@ -60,15 +60,20 @@ export class AgendaCronJobs {
 		this.reservedJobs = [];
 	}
 
-	public async add(name: string, schedule: string, callback: () => any | Promise<any>): Promise<void> {
+	public async add(name: string, schedule: string | Date, callback: () => any | Promise<any>): Promise<void> {
 		if (!this.scheduler) {
 			return this.reserve(name, schedule, callback);
 		}
 
 		this.scheduler.define(name, async () => {
 			await runCronJobFunctionAndPersistResult(async () => callback(), name);
 		});
-		await this.scheduler.every(schedule, name, {}, {});
+
+		if (typeof schedule === 'string') {
```

Comment:
I think this change in behavior should not be attached to the param type..

what about creating a new method like `addSingle` instead?

Reply:
single is the name `agenda` uses for the events that run in an interval, so I named it addAtTimestamp instead.